migrations/2024-11-25-161129_add_blurhash_to_image_details/up.sql

```diff
@@ -0,0 +1,6 @@
+-- Add a blurhash column for image_details
+ALTER TABLE image_details
+-- Supposed to be 20-30 chars, use 50 to be safe
+-- TODO this should be made not null for future versions of pictrs
```

Comment:
You said this isnt possible for proxied images, then remove the comment.

Reply:
K, I'll do that now, and check out the failures.